modules/guardduty-org-admin-account/variables.tf

```
@@ -0,0 +1,10 @@
+variable "enable" {
+  description = "(Optional) Enable monitoring and feedback reporting. Setting to false is equivalent to 'suspending'GuardDuty. Defaults to true."
+  type        = bool
+  default     = true
+}
+
+variable "guardduty_administrator_account" {
```

I think the "delegated administrator" is the AWS convention here... Makes sense to align that terminology with the variable name?
```suggestion
variable "delegated_administrator_account" {
```

---

Renamed.